src/assets/i18n/en.json5

```diff
@@ -1582,25 +1582,19 @@
 
   "cookies.consent.app.required.title": "(always required)",
 
-  "cookies.consent.app.disable-all.description": "Use this switch to enable or disable all services.",
-
-  "cookies.consent.app.disable-all.title": "Enable or disable all services",
-
```

Comment:
Just a note for other developers.  Initially, the removal of all these keys from json5 files looked confusing to me.  But, I see that you are removing keys that are no longer used in Orejime.  So, I think this cleanup is correct.